NetCoreTeamCity.Tests/Services/TestOccurencesServiceTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using FakeItEasy;
+using FluentAssertions;
+using NetCoreTeamCity.Clients;
+using NetCoreTeamCity.Exceptions;
+using NetCoreTeamCity.Models;
+using NetCoreTeamCity.Services;
+using NUnit.Framework;
+using System.IO;
+using NetCoreTeamCity.Api;
+
+namespace NetCoreTeamCity.Tests.Services
+{
+    [TestFixture]
+    public class TestOccurencesServiceTests
```

Comment:
Add more test cases - returning empty list, re-throwing exception in case of bed requests - returning null if test cases not found.